### DISCUSSION TITLE
Is there a way to update a runner's labels without deleting it and recreating it again?

### DISCUSSION THREAD

**Discussion by tking16** [actions/runner#3228] [Q&A]
It's long and annoying to do so otherwise

**Comment by Soumya-RanjanSwain** [comment]
# yes there is actually.
### go to the runner section, and select the runner that you want to use.
<img width="1097" height="230" alt="image" src="https://github.com/user-attachments/assets/3d2e9016-d8d0-4d3b-8821-82c2c27134ba" />

### you will see a section like the following:
<img width="1125" height="329" alt="image" src="https://github.com/user-attachments/assets/89d80ec5-5bc0-475c-86ce-c3cbb3c0a078" />

### click on the gear and you will be able to see option to create a label and assign it or assign labels from other runners.
<img width="1106" height="623" alt="image" src="https://github.com/user-attachments/assets/129b5191-5366-4b82-979e-982899d3b532" />

### Create one or assign one like the following
<img width="1189" height="531" alt="image" src="https://github.com/user-attachments/assets/e2aae316-25ad-4ae0-b3ce-ad608dc9f0ce" />

### and you are good to go you can use those labels in your workflows
<img width="1886" height="679" alt="image" src="https://github.com/user-attachments/assets/f6e70362-849b-43c0-8d91-5ba987ac64d6" />
